### DISCUSSION TITLE
Fun times with windows

### DISCUSSION THREAD

**Discussion by monks-dev** [Murmele/Gittyup#717] [General]
# I forked this repo at 1 am and tried to build it on windows.. 5 hours later I still haven't finished. 

![image](https://github.com/Murmele/Gittyup/assets/121908494/cdef6d7f-5fe9-4510-a35b-a93407e62be9)

**Comment by jensenr30** [comment]
![pepe trumpet whiskey blues](https://github.com/Murmele/Gittyup/assets/3083835/7bcc39f1-411b-48cb-aad9-08f2bea802a1)
